src/services/user.ts

```diff
@@ -0,0 +1,17 @@
+import { NotFoundError } from "@/types/exceptions";
+import UserDAO from "@/db/actions/user";
+import ERRORS from "@/utils/errorMessages";
+import { validateDeleteUser } from "@/utils/serviceUtils/authServiceUtil";
+
+export default class UserService {
+  static async deleteUser(userId: string) {
+    validateDeleteUser(userId);
```

Comment:
```suggestion
    validateDeleteUser({userId});
```

Shouldnt this be wrapped as an object?